File: Treasury.Application/DTOs/BudgetDto.cs
using System.Collections.Generic;

namespace Treasury.Application.DTOs;

public class BudgetDto
{
    public int Id { get; set; }

    public string NameOfClub { get; set; }

    public string FiscalYear { get; set; }

    public int NumOfItems { get; set; }

    public decimal AmountRequested { get; set; }

    public decimal AmountProposed { get; set; }

    public decimal AmountApproved { get; set; }
}

public class BudgetDetailedDto : BudgetDto
{
#nullable enable
    public List<BudgetSectionDto>? Sections { get; set; }
#nullable disable
    public bool Appealed { get; set; }

    public decimal AppealAmount { get; set; }

    public string AppealDecision { get; set; }

    public decimal ApprovedAppeal { get; set; }
}

File: src/main/java/edu/wpi/sga/treasury/application/dto/BudgetDto.java
package edu.wpi.sga.treasury.application.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@Builder
public class BudgetDto {
    private String nameOfClub;
    private Integer id;
    private String fiscalYear;
    private Integer numOfItems;
    private BigDecimal amountRequested;
    private BigDecimal amountProposed;
    private BigDecimal amountApproved;
}
